src/cmdtab.c

```diff
@@ -1087,6 +1075,16 @@ static void RedrawSwitcher(void)
 			DrawTextW(DrawingContext, title, -1, &textRect, DT_SINGLELINE | DT_CENTER | DT_BOTTOM);
 		}
 	}
+
+	// Move the original objects back since drawing has been finished
+	selectionOutline = (HPEN)SelectObject(DrawingContext, oldPen);
+	selectionBackground = (HBRUSH)SelectObject(DrawingContext, oldBrush);
+	SelectObject(DrawingContext, oldFont); // No need to delete the font, it's a stock object
+
+	// Delete the objects that we don't use any longer
+	DeleteObject(selectionOutline);
+	DeleteObject(selectionBackground);
+	DeleteObject(windowBackground);
```

Comment:
This is how I understand the docs and example code on the Microsoft pages. Also, my understanding is that it is harmless to call `DeleteObject()` for objects that are not heap-allocated. However, I wouldn't be surprised if this is still incorrect. GDI is PITA 😅